lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.cpp

```diff
@@ -1469,11 +1466,33 @@ python::runStringMultiLine(const llvm::Twine &string,
                            const PythonDictionary &locals) {
   if (!globals.IsValid() || !locals.IsValid())
     return nullDeref();
-  PyObject *result = PyRun_String(NullTerminated(string), Py_file_input,
-                                  globals.get(), locals.get());
+  PyObject *result = RunString(NullTerminated(string), Py_file_input,
+                               globals.get(), locals.get());
   if (!result)
     return exception();
   return Take<PythonObject>(result);
 }
 
+namespace lldb_private {
+namespace python {
```

Comment:
This doesn't need to be wrapped in namespaces?

Reply:
I believe it does. AFAIK you cannot implement a function declared in a namespace without explicitly qualifying it (i.e. the `using namespace` isn't enough). The compiler seems to agree with me.

Reply:
`PyObject *lldb_private::python::RunString(...) {` ?

Reply:
Do you think that's better? Note that I have more functions that'll go into this namespace.

Reply:
I don't think it's better or worse, but it is more consistent with how we've done class methods. I'll leave it up to you.

Reply:
I thought that rule was primarily aimed at anonymous namespaces, but I guess it applies here too. Regardless, it would be inconsistent which is why I didn't change the whole file. Seems like I'm the only one that prefers the namespaces so I'll change it to use the fully qualified names.